docs/content.zh/docs/dev/table/sql/show.md

```diff
@@ -411,6 +460,14 @@ table_env.execute_sql("SHOW FULL MODULES").print()
 Flink SQL> SHOW CATALOGS;
 default_catalog
 
+Flink SQL> create catalog cat2 WITH (...);
+[INFO] Execute statement succeeded.
+ 
+Flink SQL> show create catalog cat2;
+CREATE CATALOG `cat2` WITH (
+  ...
+)
+
```

Comment:
Nit: let key word be capitalized to keep consistent with other examples.

Reply:
done